asreview/webapp/authentication/models.py

```diff
@@ -0,0 +1,272 @@
+# Copyright 2019-2022 The ASReview Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime as dt
+from pathlib import Path
+from uuid import uuid4
+
+from flask_login import UserMixin
+from itsdangerous import URLSafeTimedSerializer
+from sqlalchemy import Boolean
+from sqlalchemy import Column
+from sqlalchemy import DateTime
+from sqlalchemy import ForeignKey
+from sqlalchemy import Integer
+from sqlalchemy import String
+from sqlalchemy.orm import relationship
+from sqlalchemy.orm import validates
+from sqlalchemy.orm.session import Session
+from werkzeug.security import check_password_hash
+from werkzeug.security import generate_password_hash
+
+import asreview.utils as utils
+from asreview.webapp import DB
+
+
+class User(UserMixin, DB.Model):
+    """The User model for user accounts."""
+
+    __tablename__ = "users"
+    id = Column(Integer, primary_key=True)
+    identifier = Column(String(100), nullable=False)
+    origin = Column(String(100), nullable=False)
```

Comment:
This one is related to the source of the password, right? A docstring explaining the use of each user attribute would be nice. Also something in the documentation.